topsf/tasks/inference_v2/fit_mixin.py

```diff
@@ -0,0 +1,109 @@
+# coding: utf-8
+
+import luigi
+import law
+
+from columnflow.tasks.framework.base import ConfigTask
+
+
+class FitMixin(
+    # CreateDatacards,
+    ConfigTask,  # FIXME should inherit from what?
+):
+
+    physics_model = luigi.Parameter(
+        significant=True,
+        description="The physics model to use for the fit.",
+    )
+
+    wp_name = luigi.Parameter(
+        significant=True,
+        description="Working point to perform fit for.",
+    )
+
+    fit_modes = luigi.Parameter(
+        significant=True,
+        description="Fit mode for the fit (3q:TagAndProbe,2q:ThetaLike,0o1q:ThetaLike).",
+    )
```

Comment:
Since this is essentially a mapping/dict-like structure, you could consider defining this as a `SettingsParameter`. This is defined in `columnflow.tasks.framework.parameters` and has a slightly different syntax, but would save you the trouble of parsing the string yourself.

```suggestion
    fit_modes = luigi.SettingsParameter(
        significant=True,
        description="Fit modes for each subprocess type (3q=TagAndProbe,2q=ThetaLike,0o1q=ThetaLike).",
    )
```